### PR TITLE
Use new makefile to build target, doesn't depend on go being setup.

### PR DESCRIPTION
**Pull request by brian-brazil**
Don't expect users to know what screen is.

### CODE REVIEW

File: content/docs/introduction/getting_started.md [hunk 1]
```diff
@@ -156,19 +156,15 @@ Download the Go client library for Prometheus and run three of these example
...
-cd client_golang/examples/random
-
-# Assuming a working Go setup, fetch necessary dependencies.
-go get -d
-
-# Start 3 example targets in screen sessions:
-go run main.go -listen-address=:8080
-go run main.go -listen-address=:8081
-go run main.go -listen-address=:8082
+# Start 3 example targets in the background:
```
Comment (juliusv) [inline content/docs/introduction/getting_started.md]: For users who don't know screen, backgrounding stuff could be even more confusing ("how do I stop it??"). How about just saying, "Start 3 example targets in separate terminals:"?

Comment (brian-brazil) [inline content/docs/introduction/getting_started.md]: Good point, changed.

File: content/docs/introduction/getting_started.md [hunk 2]
```diff
@@ -156,19 +156,15 @@ Download the Go client library for Prometheus and run three of these example
 processes:
 
 ```bash
-# Fetch the client library code.
+# Fetch the client library code, and compile example
```
Comment (juliusv) [inline content/docs/introduction/getting_started.md]: missing period

Comment (juliusv) [inline content/docs/introduction/getting_started.md]: (or colon)

also: no comma

Comment (brian-brazil) [inline content/docs/introduction/getting_started.md]: Done

**Pull request merged**